CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   RadioButton: fix helper text accessibility
 -   Switch: fix helper text accessibility
 -   ExpansionPanel: fix toggle button a11y using the disclosure pattern.
+-   Autocomplete: fix type of textFieldProps prop.
```

Comment:
This should be under Unreleased > Fixed and not in under 3.0.5 (probably a rebase error)